jdk/src/windows/bin/i586/jvm.cfg

```diff
@@ -30,5 +34,6 @@
 # "-XXaltjvm=<jvm_dir>" option, but that too is unsupported
 # and may not be available in a future release.
 #
--client KNOWN
--server KNOWN
+-j9vm KNOWN
```

Comment:
Please try creating this file in the closed directory (i.e. `ALT_JVMCFG_SRC := $(JDK_TOPDIR)/src/closed/$(JVMCFG_BASE)`) instead of modifying this version. It only needs to contain -j9vm

Reply:
I tried placing jvm.cfg ((with only -j9vm) at jdk/closed/windows/bin/i586 but it ended up with compilation failure as follows:
```
## Starting jdk
./jdk/make/CompileJavaClasses.gmk
CompileJavaClasses.gmk:330: *** SRC specified to SetupJavaCompilation BUILD_JDK contains missing directory /cygdrive/c/temp3/openj9-openjdk-jdk8/jdk/src/closed/share/classes.  Stop.
make[1]: *** [BuildJdk.gmk:68: classes-only] Error 2
make: *** [/cygdrive/c/temp3/openj9-openjdk-jdk8//make/Main.gmk:119: jdk-only] Error 2

./jdk/make/CompileJavaClasses.gmk
$(eval $(call SetupJavaCompilation,BUILD_JDK,\
    SETUP:=GENERATE_JDKBYTECODE,\
    SRC:=$(JDK_TOPDIR)/src/share/classes \
        $(JDK_TOPDIR)/src/$(OPENJDK_TARGET_OS_API_DIR)/classes \
        $(MACOSX_SRC_DIRS) \
        $(AIX_SRC_DIRS) \
        $(JDK_OUTPUTDIR)/gensrc \
        $(JDK_OUTPUTDIR)/gensrc_no_srczip \
        $(CLOSED_SRC_DIRS),\  <---------------

ifndef OPENJDK
  CLOSED_SRC_DIRS := $(JDK_TOPDIR)/src/closed/share/classes \
      $(wildcard $(JDK_TOPDIR)/src/closed/$(OPENJDK_TARGET_OS_API_DIR)/classes)
endif

./make/common/JavaCompilation.gmk
  # Handle addons and overrides.
  $1_SRC:=$$(call ADD_SRCS,$$($1_SRC))
  # Make sure the dirs exist.
  $$(foreach d,$$($1_SRC), $$(if $$(wildcard $$d),,$$(error SRC specified to SetupJavaCompilation $1 contains missing directory $$d))) <----------
```
The error above shows it complains when the "closed" folder exists. 
The compilation passed only when the "closed" folder was totally removed.

Reply:
I think we'll leave further "improvement" to future work. The existing solution that changes this file is no worse than all the other modified jvm.cfg files.